axelrod/tests/unit/test_moran.py

```diff
@@ -330,6 +330,21 @@ def test_iter(self):
         mp = MoranProcess((p1, p2))
         self.assertEqual(mp.__iter__(), mp)
 
+    def test_population_plot(self):
+        # Test that can plot on a given matplotlib axes
+        axelrod.seed(15)
+        players = [random.choice(axelrod.demo_strategies)() for _ in range(5)]
+        mp = axelrod.MoranProcess(players=players, turns=30)
+        mp.play()
+        fig, axarr = plt.subplots(2, 2)
+        ax = axarr[1, 0]
+        mp.populations_plot(ax=ax)
+        self.assertEqual(ax.get_xlim(), (-0.8, 16.8))
+        self.assertEqual(ax.get_ylim(), (0, 5.25))
+
+        # Run with a give axis
+        mp.populations_plot()
```

Comment:
Can we test something. 

Perhaps (to avoid adding to the previous plot):

```python
plt.figure()
self.assertl(mp.populations_plot(), matplotlib.pyplot.Figure)  # I'm guessing?
```